Detectors/PHOS/workflow/src/CellConverterSpec.cxx

```diff
@@ -37,10 +37,12 @@ void CellConverterSpec::init(framework::InitContext& ctx)
 
 void CellConverterSpec::run(framework::ProcessingContext& ctx)
 {
-  LOG(debug) << "[PHOSCellConverter - run] called";
-  auto dataref = ctx.inputs().get("digits");
-  auto const* phosheader = o2::framework::DataRefUtils::getHeader<o2::phos::PHOSBlockHeader*>(dataref);
-  if (!phosheader->mHasPayload) {
+  // LOG(debug) << "[PHOSCellConverter - run] called";
+  // auto dataref = ctx.inputs().get("digits");
+  // auto const* phosheader = o2::framework::DataRefUtils::getHeader<o2::phos::PHOSBlockHeader*>(dataref);
+  // if (!phosheader->mHasPayload) {
+  auto digits = ctx.inputs().get<std::vector<o2::phos::Digit>>("digits");
+  if (!digits.size()) { // nothing to process
```

Comment:
@sevdokim , what if we have event without digits? Probably it is better to use TriggerRecords - they will be non-zero in such events

Reply:
@peressounko can such case even be?

Reply:
but yes, I agree if we have digit but no corresponding trigger record that can be a problem